### ISSUE TITLE
`getAllContacts` returns some contacts multiple times

### ISSUE THREAD

**Issue by KishanBagaria** [codebytere/node-mac-contacts#13]
If you run the following, you might see some contacts multiple times:
```js
const ac = require('node-mac-contacts').getAllContacts()
const seenIDs = new Set()
for (const c of ac) {
  if (seenIDs.has(c.identifier)) {
    console.log(c)
  }
  seenIDs.add(c.identifier)
}
```

It returns my contact 3 times and Apple's contact 2 times:

<img width="844" alt="" src="https://user-images.githubusercontent.com/1093313/80368805-f1287600-88aa-11ea-9429-c56bec44e3f7.png">

Not sure if this is a quirk of the native API or something to do with multiple containers/unified contacts. The duplicate contacts are deeply equal.

**Comment by codebytere** [comment]
@KishanBagaria this is likely a problem at the Apple level but i think it'd be possible to do some pre-emptive filtration to compensate for it 🤔 fwiw this doesn't happen on my machine, interestingly.

**Comment by KishanBagaria** [comment]
Figured out why it's happening.

I have four containers and my contact is present in three and Apple's contact is present in two:

<img width="494" alt="" src="https://user-images.githubusercontent.com/1093313/80925130-10436e00-8dab-11ea-8fcb-a055ff361ff3.png">

Since we're looping over each container and fetching a unified contact, the contact is the same.

To repro: you can add a Google account in System Preferences -> Internet Accounts and tick Contacts, assuming you have a "me" contact in both Google and iCloud.

**Comment by codebytere** [comment]
Ah that’d explain it! I don’t have any containers set up like that so it makes sense i couldnt repro - I’ll try to get a fix up for this at some point soon!

**Comment by codebytere** [comment]
Closed in e6c13fe.
